bin/opm

```diff
@@ -1962,7 +1962,7 @@ sub init_installation_ctx ($) {
         make_path $manifest_dir;
     }
 
-    my $resty = File::Spec->catfile($install_dir, "bin", "resty");
+    my $resty = File::Spec->catdir($FindBin::RealBin, "resty");
```

Comment:
Here we should test "$install_dir/bin" and "$install_dir/../bin" with the `-f` operator instead. Using `FindBin` is wrong because `resty` may not be installed in the same directory as the current `opm` script.